OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs

```diff
@@ -541,6 +546,7 @@ private async void PlayPreviousVideo()
 
             MediaWrapper.CachedDriveItem = App.Current.MediaItemList[--MediaListIndex];
             await PlayMedia();
+            fileNameOverlayTimer.Start();
```

Comment:
Should this be here?

Reply:
Yes, unless we simplify it even more by putting this logic around the starttime check in `PlayMedia()`.

Reply:
Yes that sounds better to me.